R/quote.R

```diff
@@ -100,6 +106,9 @@ setGeneric("dbQuoteIdentifier",
 
 quote_identifier <-
   function(conn, x, ...) {
+    if (is.list(x)) {
```

Comment:
I think auto-vectorisation over lists is best avoided because it violates type-stability (and we generally avoid it elsewhere)